analyzers/build/build.go

```diff
@@ -0,0 +1,125 @@
+package build
```

Comment:
Docker related code should be inside a `docker` package. That package can contain logic related to login, build and publish.